pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -825,11 +825,11 @@ synchronized void playLeader() {
         boolean becameFollower = false;
         while (!Thread.currentThread().isInterrupted()) {
             try {
+                initWaiter.await();
                 if (!serviceUnitStateChannel.isChannelOwner()) {
                     becameFollower = true;
                     break;
                 }
-                initWaiter.await();
```

Comment:
No. It is a minor fix. Otherwise, isChannelOwner will likely fail for the first time before fully initiating the channel.  (It will retry anyway, but we don't want to spam with error logs when starting the broker)